Frontend/src/components/Input/PopupFormular.svelte

```diff
@@ -1,166 +1,175 @@
 <script>
-    import Checkbox from "svelte-checkbox";
-    import SelectInput from "./SelectInput.svelte";
-    import InputGroup from "./InputGroup.svelte";
-    import DateInput from "./DateInput.svelte";
-    import TextInput from "./TextInput.svelte";
-    import AutocompleteInput from "./AutocompleteInput.svelte";
-    import InputTypes from "./InputTypes";
-    import { keyValueStore } from "../../utils/stores";
-    import { createEventDispatcher } from "svelte";
+  import Checkbox from "svelte-checkbox";
```

Comment:
I sometimes have the feeling that VSCode can't get it right and adds/removes whitespaces in a mood :D